src/oraWrapper.ts

```diff
@@ -0,0 +1,24 @@
+import * as ora from "ora";
```

Comment:
Not sure this is necessary if you move the spinner to the command as per above and leave the unit-testable area free of spinners.

Reply:
Removed `oraWrapper`